spec.md

```diff
@@ -338,6 +338,11 @@ defines a "CLI-friendly" string as a short string that MUST only use lowercase
 characters, numbers and hyphens, with no spaces. This will make it easier for
 users when they have to type it as an argument on the command line.
 
+For backwards compatibility reasons, this specification does not preclude
```

Comment:
Should this read "CLI-unfriendly"?

Reply:
I think it can go either way, but "unfriendly" does has kind of a funny tweak to it so I'll switch it.